dcos/util.py

```diff
@@ -547,3 +547,19 @@ def io_exception(path, errno):
 
 
 logger = get_logger(__name__)
+
+
+def get_proxy_dict_from_env():
+    """ Returns dict with proxy parameters
+
+    :returns: Dict with proxy parameters
+    :rtype: dict
+    """
+
+    proxies = dict()
+
+    for name, value in os.environ.items():
+        name = name.lower()
+        if value and name[-6:] == '_proxy':
```

Comment:
Let's be more explicit on the comparison. Easier on the reader later. Can we match for only http_proxy and https_proxy? Thanks!